app/views/shared/_additional_expense_form.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class="nested-form-wrapper">
+    <li>
+      <div class="expense-container row" data-index="<%= form.index %>">
+        <div class="input-style-1 col-sm-3">
+          <h5><%= form.label :additional_expense, "a. Expense amount" %></h5>
+          <div class="input-group">
+            <span class="dollar-sign">
+              <%= form.number_field :other_expense_amount,
+                placeholder: "0",
+                class: "form-control other-expense-amount",
+                min: "0", max: 1000, step: 0.01 %>
```

Comment:
After saving and coming back to edit, I get an expense in the form with only one cent place. 
<img width="1107" alt="Screenshot 2024-07-23 at 7 01 16 AM" src="https://github.com/user-attachments/assets/6ea27c3d-870b-4fe1-abb5-08e793d6f4e0">

Reply:
@schoork added number_with_precision to the field, let me know if this display is ok:
<img width="953" alt="Screen Shot 2024-07-23 at 1 12 11 PM" src="https://github.com/user-attachments/assets/d7e4648a-ab1d-49ff-9db7-1f43e2fa1bb4">